spec/support/umm_s_forms/contact_information_form.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+
+shared_examples_for 'Contact Information Form' do
```

Comment:
There's no Contact Information form... are these already in the shared assertions file, or maybe should be moved there?

Reply:
Ah yeah, old code I didn't use